### PR TITLE
Update dependencies and bundler config

### PR DESCRIPTION
**Pull request by jamesrweb**
Prepares dependencies and bundler configurations for the next version release.

### CODE REVIEW

File: src/components/ReactP5WrapperWithSketch.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { createRef, useEffect, useRef } from "react";
+import React, { useEffect, useRef } from "react";
```
Comment (yevdyko) [inline src/components/ReactP5WrapperWithSketch.tsx:2]: We don't use "createRef", so it can be gone, right?

**Review by yevdyko — APPROVED** [review]
👏

File: src/components/ReactP5WrapperWithSketch.tsx (hunk 1 above)
Comment (jamesrweb) [reply src/components/ReactP5WrapperWithSketch.tsx:2]: It's gone 😉!

Comment (jamesrweb) [reply src/components/ReactP5WrapperWithSketch.tsx:2]: @yevdyko can you re-approve the latest changes and resolve this conversation? 😄

**Review by yevdyko — APPROVED** [review]
🚢

**Pull request merged**